xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -581,6 +581,9 @@ void CDVDDemuxFFmpeg::Reset()
 
 void CDVDDemuxFFmpeg::Flush()
 {
+  if (m_ioContext)
```

Comment:
there is only one io context. either we create it an assign it to m_pFormatContext->pb or ffmpeg creates it. 
see https://github.com/afedchin/xbmc/blob/6596aa1901dcde54f7dfe6efba3de94940466cfb/xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp#L385